apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -0,0 +1,260 @@
+/**
+ * Generated by `createservice meter.RegisterMetersReadingsService --type mutations`
+ */
+const dayjs = require('dayjs')
+const { get, isUndefined, isEmpty, isNumber, isString } = require('lodash')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, find, getByCondition, getById } = require('@open-condo/keystone/schema')
+
+const { PropertyResolver } = require('@condo/domains/billing/schema/resolvers')
+const access = require('@condo/domains/meter/access/RegisterMetersReadingsService')
+const { IMPORT_CONDO_METER_READING_SOURCE_ID } = require('@condo/domains/meter/constants/constants')
+const {
+    TOO_MUCH_READINGS,
+    ORGANIZATION_NOT_FOUND,
+    PROPERTY_NOT_FOUND,
+    INVALID_METER_VALUE,
+} = require('@condo/domains/meter/constants/errors')
+const { Meter, MeterReading } = require('@condo/domains/meter/utils/serverSchema')
+
+const READINGS_LIMIT = 500
+
+const ERRORS = {
+    TOO_MUCH_READINGS: (sentCount) => ({
+        code: BAD_USER_INPUT,
+        type: TOO_MUCH_READINGS,
+        message: `Too much readings. Maximum is ${READINGS_LIMIT}.`,
+        messageForUser: 'api.meter.registerMetersReadings.TOO_MUCH_READINGS',
+        messageInterpolation: { limit: READINGS_LIMIT, sentCount },
+    }),
+    ORGANIZATION_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: ORGANIZATION_NOT_FOUND,
+        message: 'Organization not found',
+        messageForUser: 'api.meter.registerMetersReadings.ORGANIZATION_NOT_FOUND',
+    },
+    PROPERTY_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: PROPERTY_NOT_FOUND,
+        message: 'Property not found',
+        messageForUser: 'api.meter.registerMetersReadings.PROPERTY_NOT_FOUND',
+    },
+    INVALID_METER_VALUE: (key, value) => ({
+        code: BAD_USER_INPUT,
+        type: INVALID_METER_VALUE,
+        message: 'Invalid meter value',
+        messageForUser: 'api.meter.registerMetersReadings.INVALID_METER_VALUE',
+        messageInterpolation: { key, value },
+    }),
+}
+
+function toISO (str) {
+    return dayjs(str).toISOString()
+}
+
+/**
+ * @param value
+ * @return {undefined|string|null}
+ */
+function normalizeMeterValue (value) {
```

Comment:
Should it be part of `resolveInput` and `validateInput` of corresponding model? Since it possible to create is not using this model